docs/tutorial.rst

```diff
@@ -227,8 +240,7 @@ You can save this as a JSON file, e.g., tsp_example.json, and then call the fram
 
     python src/main.py --modules tsp_example.json
 
-Exploring a Problem in Jupyter Notebook
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+License
```

Comment:
Same as for the paper part. We mention the license on the welcome page because we think it is not required as part of the tutorial.